doc/source/release-notes/Percona-Server-8.0.17-8.rst

```diff
@@ -0,0 +1,86 @@
+.. rn:: 8.0.17-8:
+
+================================================================================
+|Percona Server| |release|
+================================================================================
+
+|Percona| announces the release of |Percona Server| |release| on |date|
+(downloads are available `here
+<https://www.percona.com/downloads/Percona-Server-8.0/>`__ and from the
+`Percona Software Repositories
+<https://www.percona.com/doc/percona-server/8.0/installation.html#installing-from-binaries>`__).
+
+This release includes fixes to bugs found in previous releases of |Percona
+Server| 8.0.
+
+|Percona Server| |release| is now the current GA release in the 8.0 series. All
+of |Percona|’s software is open-source and free.
+
+Percona Server for MySQL 8.0 includes all the `features available in MySQL
+8.0.17 Community Edition
+<https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-17.html>`__ in
+addition to enterprise-grade features developed by Percona.
+
+New Features
+================================================================================
+
+|Percona Server| has implemented the ability to have a |MySQL| `user
+<https://www.percona.com/doc/percona-server/8.0/management/utility_user.html>`__
+who has system access to do administrative tasks but limited access to user
+schemas. The user is invisible to other users. This feature is especially
+useful to those who are operating |MySQL| as a Service. This feature has the
+same functionality as the utility user in earlier versions and has been
+delay-ported to version 8.0.
+
+|Percona Server| has implemented `data masking and de-identification functions
+<https://www.percona.com/doc/percona-server/8.0/security/data-masking.html>`__.
+
```

Comment:
same as utility user, we should use :ref:`data-masking_plugin` instead of external link so the link is always up to date when docs are rebuilt. 

This will require dm_plugin page to exist.

Reply:
Updated.